pytorch_toolkit/action_recognition_2/model_templates/gesture-recognition/s3d-rgb-mobilenet-v3-stream-jester/template.yaml

```diff
@@ -0,0 +1,58 @@
+name: s3d-rgb-mobilenet-v3-stream-jester
+domain: Action Recognition 2
+problem: Gesture Recognition
+framework: OTEAction v0.6.0
+summary: ASL Recognition based on S3D-MobileNet-V3.
+annotation_format: CustomAction
+dependencies:
+- sha256: 387f42e4a71ca0955e174d4661f795989b35056993f0049f45df4c56f83273d4
+  size: 33272762
+  source: https://docs.google.com/uc?export=download&id=1lDm2qOxMRyXZW6y7owlQBv8SGvGIKpcX
+  destination: snapshot.pth
+- source: ../../../../../pytorch_toolkit/ote/tools/train.py
+  destination: train.py
+- source: ../../../../../pytorch_toolkit/ote/tools/eval.py
+  destination: eval.py
+- source: ../../../../../pytorch_toolkit/ote/tools/export.py
+  destination: export.py
+- source: ../../../../../pytorch_toolkit/ote/tools/quantize.py
+  destination: quantize.py
+- source: ../../../../../pytorch_toolkit/ote
+  destination: packages/ote
+- source: ../../requirements.txt
+  destination: requirements.txt
+max_nodes: 1
+training_target:
+- GPU
+inference_target:
+- CPU
```

Comment:
does it work on all listed devices?

Reply:
Is the target section about the original model format (.pth) or converted one (ie)?

Reply:
Good question actually. There is no usage in our OTE. But iGPU, VPU stands for our devices. And yes, as far as I know this is for ie.

Reply:
Changed to `CPU` and `iGPU` only.